pos_order_mgmt/models/pos_order.py

```diff
@@ -131,11 +131,17 @@ def _prepare_done_order_for_pos(self):
     @api.multi
     def _prepare_done_order_line_for_pos(self, order_line):
         self.ensure_one()
+        lot_name = ''
+        for data in order_line.pack_lot_ids:
+            lot_name += data.lot_name + ','
+        if lot_name:
+            lot_name = lot_name[:-1]
         return {
             'product_id': order_line.product_id.id,
             'qty': order_line.qty,
             'price_unit': order_line.price_unit,
             'discount': order_line.discount,
+            'lot_name': lot_name,
```

Comment:
Isn't it possible to send a list/array?

I mean, you're concatenating and then splitting in js.
Why not just send this?

`


```suggestion
            'lot_name': order_line.pack_lot_ids.mapped('lot_name'),
```

Reply:
It'd also be good to call it `pack_lot_names`